config/config_test.go

```diff
@@ -56,7 +56,7 @@ commit-timeout="41s"`)
 	c.Assert(conf.BinlogSocket, Equals, "/tmp/socket")
 
 	// Test that the value will be overwritten by the config file.
-	c.Assert(conf.Performance.RetryLimit, Equals, uint(10))
+	c.Assert(conf.Performance.RetryLimit, Equals, uint32(10))
```

Comment:
I have same concern with Yuxing. Why do we change `unit` to `unit32`? The only reason I can come up with is `uint32`  need less space. Is my guess be the reason?